hazelcast/src/main/java/com/hazelcast/multimap/MultiMap.java

```diff
@@ -318,6 +318,20 @@
     @Nonnull
     UUID addLocalEntryListener(@Nonnull EntryListener<K, V> listener);
 
+    /**
+     * Adds a local entry listener for this multimap.
+     * <p>
+     * The listener will be notified for all multimap
```

Comment:
the javadoc is not correct, the listener will not be notified for `all` multimap events but only events of locally owned entries. I suggest you to link the above method like `{@link #addLocalEntryListener(EntryListener)}` and just mention the difference, e.g ability to configure if the event should contain the value.

Also it would be good to mention in above method's javadoc that the events does not contain the value, and direct the reader to this method via another link.

Reply:
Sure, added tests and updated javadoc.